Doc/library/socket.rst

```diff
@@ -752,6 +752,9 @@ The :mod:`socket` module also offers various network-related services:
    For IPv6 addresses, ``%scope`` is appended to the host part if *sockaddr*
    contains meaningful *scopeid*. Usually this happens for multicast addresses.
 
+   For more information about *flags* you can consult the
+   `manpage <http://man7.org/linux/man-pages/man3/getnameinfo.3.html>`_.
```

Comment:
You could add the name of the manpage so people prefering to consult it locally can easily do.
```suggestion
   `getnameinfo manpage <http://man7.org/linux/man-pages/man3/getnameinfo.3.html>`_.
```